.gitlab-ci.yml

```diff
@@ -275,7 +279,8 @@ run-certification-tests:
     DEVICE_UNDER_TEST: prplmesh
   script:
       - |
-        if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
+        if [ "$DEVICE_UNDER_TEST" = netgear-rax40 ] ; then
+            tools/deploy_firmware.py --device netgear-rax40 --target-name "$DEVICE_UNDER_TEST" --image NETGEAR_RAX40-squashfs-fullimage.img
```

Comment:
Now I see this, I'm thinking that it would be nicer to have a .deploy-to-target template with a before_script that:
- kills all targets;
- deploys firmware;
- deploys ipk.

Because I think the certification setup would also benefit from killing the other targets.

(This and my other comments to this file are probably for a follow-up PR).

Reply:
I agree on both comments. My secret plan was to get rid of `run-certification-tests` and to also use a single job per test for the nightly tests.

It requires some more work though, so as you said I'll keep it for another PR.